src/components/AddManualBlocks.js

```diff
@@ -3,7 +3,7 @@ import { makeStyles } from "@material-ui/core/styles";
 import Modal from "@material-ui/core/Modal";
 import { connect } from "react-redux";
 import { axiosWithAuth } from "./AxiosWithAuth";
-import DemomanualBlocks from "./Modal_Components/demomanualBlocks";
+import ManualTransaction from "./Modal_Components/ManualTransaction";
```

Comment:
Good job at renaming here, it's good to be consistent with naming. This is more semantic.